services/client/src/components/video/content.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./styles/content";
+import TimeToVideo from "./time-to-video";
+
+const formatTitle = str =>
+  str.toLowerCase().replace(/\b(\w)/g, s => s.toUpperCase());
```

Comment:
This functionality exists as part of `components/event/content.js` - worth pulling it out, or brining in something like lodash and using [`_.startCase`](https://lodash.com/docs/4.17.10#startCase)

Reply:
yes; I'll rework this using lodash and `startCase`.